src/Keboola/StorageApi/Client.php

```diff
@@ -1851,13 +1862,19 @@ public function apiPostMultipart($url, $postData = null, $handleAsyncTask = true
      *
      * @param string $url
      * @param array $postData
-     * @return mixed|stringgit d
+     * @param string|null $responseFileName
+     * @param bool $handleAsyncTask
+     * @return mixed|string
      */
-    public function apiPut($url, $postData = null)
+    public function apiPut($url, $postData = null, $responseFileName = null, $handleAsyncTask = true)
```

Comment:
Bylo to potřeba měnit? Myslím si že by nemělo.

Reply:
Jasně už tomu rozumím, jde o ten response kód. Ten jsem tam dal špatný viž. backend.